submodules/LIB/hardware/syn/build.tcl

```diff
@@ -25,7 +25,7 @@ set_db stdout_log genus.log
 set_db information_level 7
 set_db super_thread_debug_directory st_part_log
 
-set_db hdl_error_on_latch true 
+set_db hdl_error_on_latch false 
```

Comment:
> You can't disable this for us! We need this!
> 
> You need another if statement: check if the submodules/PICORV32 directory exists and disable the check.
> 

That is a very good solution, will be implemented

Reply:
Because the setup dir is not an argument passed to the tcl file, he checks all the filenames (without the path) in $VSRC, and if picorv32.v is found, the error on latches is disabled.

Reply:
not acceptable; the script knows exactly where picorv32.v 


Jut ask the team how to do this

Reply:
OK will fix

Reply:
It checks using relative paths if the submodule PICORV32 exists. If it exists, it will pass a flag so the TCL script will allow latches. An important thing: because the setup directory doesn't exist on the server (after the SSH), I gave it different names. This is because when the Makefile runs in the build directory locally it will check the setup dir because it actually exists, Inside the server the setup dir never exists because only the build dir is copied.

Reply:
setupdir does not exist for external users ; just check if ../src/picorv32.v exists